docs/authentication.md

```diff
@@ -32,3 +32,84 @@ spec:
     rbac: {}
 ```
 
+## Heptio Authenticator for AWS
+
+If you want to turn on Heptio Authenticator for AWS, you can add this block 
+to your cluster:
+
+```
+authentication:
+  heptio: {}
+```
+
+For example:
+
+```
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  name: cluster.example.com
+spec:
+  authentication:
+    heptio: {}
+  authorization:
+    rbac: {}
+```
+
+Once the cluster is up you will need to create the heptio authenticator
+config as a config map. (This can also be done when boostrapping a cluster using addons)
+For more details on heptio authenticator please visit (heptio/authenticator)[https://github.com/heptio/authenticator]
+Example config:
+
+```
+---
+apiVersion: v1
```

Comment:
What happen's in the config isn't there? .. does it's just crashloop the pods? ... Just wondering in case it effects rollouts .. i'm pretty sure the validate cluster code looks for pod restarts in kube-system ..

Reply:
The pod will sit in a `creating` status. I have not tired doing a rollout while it's in this state, but will do so.

Reply:
Ar yes! .. it would be stuck in pending .. Still if you can quickly verify / confirm it handles on a pre-built cluster and can rollout, that would be cool

Reply:
The pod stays in a in state:
kube-system heptio-authenticator-aws-4fkzp 0/1 ContainerCreating 0 10m

with the error:
Unable to mount volumes for pod "heptio-authenticator-aws-4fkzp_kube-system(93f46fcb-654e-11e8-a014-0e21d0512306)": timeout expired waiting for volumes to attach or mount for pod "kube-system"/"heptio-authenticator-aws-4fkzp". list of unmounted volumes=[config]. list of unattached volumes=[config output state default-token-6rtmz]

However I'm still able to perform a full cluster rolling update while the cluster is in this state so we should be ok.

Reply:
much appreciated :+1: